index.html

```diff
@@ -22,93 +27,92 @@
-  * **Spring Data MongoDB** - Azure MongoDB (via [Azure CosmosDB](https://azure.microsoft.com/services/cosmos-db/))* **Spring Data MySQL** - [Azure Database for MySQL](https://azure.microsoft.com/services/mysql/) (For an example, see [Build a Java and MySQL web app in Azure](https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-java-mysql).)	
-  * **Spring Data for Apache Cassandra** - Azure Cassandra (via [Azure CosmosDB](https://azure.microsoft.com/services/cosmos-db/))	
-  * **Spring Data Redis** - [Azure Redis Cache](https://azure.microsoft.com/services/cache/) (For an example, see [Configure a Spring Boot Initializer app to use Redis in the cloud with Azure Redis Cache](https://docs.microsoft.com/java/azure/spring-framework/configure-spring-boot-initializer-java-app-with-redis-cache).)
- 
-* Additional cloud-computing features :
-   * Azure App Service - [Quickstart: Create a Java web app in App Service on Linux](https://docs.microsoft.com/azure/app-service/containers/quickstart-java)
-   * Container - [Deploy a containerized Spring Boot app to Azure](https://docs.microsoft.com/java/azure/spring-framework/deploy-containerized-spring-boot-java-app-with-maven-plugin)
-   * Kubernetes - [Deploy a Spring Boot Application on a Kubernetes Cluster in the Azure Container Service](https://docs.microsoft.com/java/azure/spring-framework/deploy-spring-boot-java-app-on-kubernetes)
-  <br><br>
-* Additional tooling support:
-   * [Spring Boot Extension Pack for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Pivotal.vscode-boot-dev-pack) – For more information, see [Build Spring Boot Apps with VS Code](https://code.visualstudio.com/docs/java/java-spring-boot).
+## Spring Cloud Features
 
-<span id="quick-start"></span>
+* Spring Cloud Stream Binder for [Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) (native API and Apache Kafka API), including Spring Integration Channel Adapters
+* Spring Cloud Stream Binder for [Azure Service Bus](https://azure.microsoft.com/services/service-bus/) Topic and Queue, including Spring Integration Channel Adapters
+* Spring Resource Abstraction for [Azure Storage](https://azure.microsoft.com/services/storage/)
+* Spring Cache auto-configuration for [Azure Cache for Redis](https://azure.microsoft.com/services/cache/)
+* Spring Cloud Config for [Azure App Configuration](https://docs.microsoft.com/azure/azure-app-configuration/) (Preview)
 
-## Quick Start
+## Spring Boot Support
+
+There are four entries in Spring Initializr for Spring Cloud Azure:
+* Azure support
+* Spring Boot Starter for [Azure Active Directory](https://azure.microsoft.com/services/active-directory/)
+* Spring Boot Starter for [Azure Key Vault](https://azure.microsoft.com/services/key-vault/) 
+* Spring Boot Starter for [Azure Storage](https://azure.microsoft.com/services/storage/)
+
+You can get more from Github:  
```

Comment:
Should this sentence be different? None of the URLs in the following list point to GitHub.